tests/test_commands.py

```diff
@@ -1655,7 +1655,10 @@ def create_file_versions(project: models.Project):
 
         # Error email should be sent
         assert len(outbox1) == 1
-        assert "[INVOICING CRONJOB] <ERROR> Error in monthly-usage cronjob" in outbox1[-1].subject
+        assert (
+            "[INVOICING CRONJOB] (DEVELOPMENT) <ERROR> Error in monthly-usage cronjob"
+            in outbox1[-1].subject
+        )
```

Comment:
Change the `in` to `==` in all of these instances (or if `==` doesn't work in this case, find another solution), at the moment the code added lets the test pass, but it should fail since we don't want `(DEVELOPMENT)` at the end of the subject.  
```suggestion
        assert outbox1[-1].subject == "[INVOICING CRONJOB] (DEVELOPMENT) <ERROR> Error in monthly-usage cronjob"
```